### DISCUSSION TITLE
Lesson 6: Gitpod use

### DISCUSSION THREAD

**Discussion by yeahChibyke** [Cyfrin/foundry-full-course-cu#415] [Q&A]
I use a windows system, and despite all I have tried, I am unable to install WSL on it. I have to use Gitpod. But so far in the video, there is no explanation on how to go about 'implementing' Foundry on Gitpod. Is that(implementing Foundry on Gitpod) even a thing? I have checked online and asked chatGPT, but I have not gotten any useful answers. Please, I need help on how to use Foundry with Gitpod.

**Comment by ghost** [comment]
Try using [Git Bash](https://git-scm.com/downloads) as an alternative for WSL. You can use Git Bash as an alternative to Windows Subsystem for Linux (WSL) for certain tasks. While they are not exactly the same, they share some similarities in providing a command-line environment on Windows

**Reply by yeahChibyke** [reply]
I installed Foundry directly in Gitpod, and I am currently running it. It has not given any issues so far. Fingers crossed.

**Reply by ghost** [reply]
Got it 👍
